build_debian.sh

```diff
@@ -188,7 +188,20 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     usbutils                \
     pciutils                \
     iptables-persistent     \
-    logrotate
+    logrotate               \
+    grub2-common
```

Comment:
Checked again - it was failing because of tabulate package (removed grub2-common from this line)